src/components/RepoBranchesDrawer/RepoBranchesDrawer.tsx

```diff
@@ -1,47 +1,58 @@
 import React from "react";
 
+import Loader from "@components/Loader";
 import GitHubStore from "@store/GitHubStore";
 import { BranchesItem } from "@store/GitHubStore/types";
 import { Drawer } from "antd";
+import { useParams } from "react-router-dom";
+import useReposListContext from "utils/useReposListContext";
 
 type RepoBranchesDrawerProps = {
   onClose: () => void;
   visible: true | false;
-  owner: string;
-  repo: string;
 };
 
 const RepoBranchesDrawer: React.FC<RepoBranchesDrawerProps> = ({
   onClose,
   visible,
-  repo,
-  owner,
 }: RepoBranchesDrawerProps) => {
   const [branches, setBranches] = React.useState<BranchesItem[] | null>(null);
   const [load, setLoad] = React.useState(false);
   const [gitHubStore] = React.useState(() => new GitHubStore());
-
-  if (!visible && load) setLoad(false);
+  const { company, title } = useParams();
+  const { context } = useReposListContext();
+  if (title && company) visible = true;
```

Comment:
можно вынести в переменную 
```suggestion
  const visible = !!title && !!company;
```